flight/tasks/__init__.py

```diff
@@ -0,0 +1,207 @@
+import abc
+
+from simple_pid import PID
+from dronekit import VehicleMode
+
+from flight import constants as c
+from flight import flightconfig as f
+from flight.drone.exceptions import EmergencyLandException
+
+KP = 1
```

Comment:
we need a comment for each of the constants in this group indicating what they are/what their purpose is.